apis/cloud.redhat.com/v1alpha1/clowdenvironment_types.go

```diff
@@ -55,8 +57,7 @@ type MetricsMode string
 // metrics services and their probes.
 type MetricsConfig struct {
 	// The port that metrics services inside ClowdApp pods should be served on.
-	// If omitted, defaults to 9000.
-	Port int32 `json:"port,omitempty"`
+	Port int32 `json:"port"`
```

Comment:
I feel like we talked about shuffling these port attributes around, but I don't remember specifics.  Did we agree to keep the metrics port in a separate provider?  Feels weird looking at it here.

Reply:
We did - but we can change it back - though that would mean some more backwards compat stuff - I'd prefer to move it back I think - but it would mean changing the spec....again

Reply:
Eh, let's just leave it.  Hopefully it will feel more natural when we have more config added to the metrics provider.